gtk/src/makedeps.py

```diff
@@ -65,41 +66,35 @@ def main():
     try:
         depsfile = open("widget.deps", "w")
     except Exception as err:
-        print >> sys.stderr, ( "Error: %s"  % str(err) )
+        print("Error: %s" % str(err), stream=sys.stderr)
```

Comment:
Augh! Apparently neither you nor I hit an exception in our testing of this particular code, @bradleysepos, as that should be

```python
print("Error: %s" % str(error), file=sys.stderr)
```
I only thought to check because I do that roughly _half_ the time, sigh. I'll submit a followup PR to fix, sorry about that.

Reply:
Yep, I didn't see it and didn't hit it in testing either. 🤦🏻‍♂️ It happens. Will merge your follow-up when I see it.

Reply:
Just opened as #4741, thanks.